scripts/install/run.sh

```diff
@@ -0,0 +1,337 @@
+#!/usr/bin/env bash
+set -e
+
+# Prints header
+function printh {
+    local input="$1"
+    local len=$((${#1}+2))
+    printf "\033[32m" # start green color
+    printf "\n+"
+    printf -- "-%.0s" $(seq 1 $len)
+    printf "+\n| $input |\n+"
+    printf -- "-%.0s" $(seq 1 $len)
+    printf "+\n\n"
+    printf "\033[0m" # end green color
+}
+
+function integrate_with_smartedit {
+  cmd_install "$@"
+
+  cp ${CLONE_DIR}/projects/storefrontapp/src/webApplicationInjector.js ${APP_DIR}/src
+
+  jq ".projects.${APP_NAME}.architect.build.options.assets += [\"src/webApplicationInjector.js\"]" ${APP_DIR}/angular.json > ${APP_DIR}/temp.json
+
+  mv ${APP_DIR}/temp.json ${APP_DIR}/angular.json
+}
+
+function delete_dir {
+    if [ -d "${1}" ]; then
```

Comment:
Assigning the first argument to a variable makes the function easier to change if needed